Traceback (most recent call last):
  File "run.py", line 170, in <module>
    tenOutput = estimate(tenOne, tenTwo)
  File "run.py", line 135, in estimate
    netNetwork = Network().cuda().eval()
  File "run.py", line 96, in __init__
    self.load_state_dict({ strKey.replace('module', 'net'): tenWeight for strKey, tenWeight in torch.hub.load_state_dict_from_url(url='http://content.sniklaus.com/github/pytorch-spynet/network-' + arguments_strModel + '.pytorch', file_name='spynet-' + arguments_strModel).items() })
TypeError: load_state_dict_from_url() got an unexpected keyword argument 'file_name'
